googletest/include/gtest/internal/gtest-port.h

```diff
@@ -920,18 +920,22 @@ using ::std::tuple_size;
 #endif  // GTEST_HAS_SEH
 
 #ifdef _MSC_VER
-
 # if GTEST_LINKED_AS_SHARED_LIBRARY
 #  define GTEST_API_ __declspec(dllimport)
 # elif GTEST_CREATE_SHARED_LIBRARY
 #  define GTEST_API_ __declspec(dllexport)
 # endif
-
-#endif  // _MSC_VER
-
-#ifndef GTEST_API_
-# define GTEST_API_
-#endif
+#elif __GNUC__ >= 4 || defined(__clang__)
+# if GTEST_CREATE_SHARED_LIBRARY
```

Comment:
I'm not sure I fully understand. Are you proposing to get rid of the `#if GTEST_CREATE_SHARED_LIBRARY` because the `__attribute__((visibility ("default")))` doesn't have an effect if we're building a static library? I think that would be a valid simplification. I'll try that.